icubam/predicu/data.py

```diff
@@ -4,23 +4,55 @@
 import os
 import pickle
 import urllib.request
+from typing import Dict
 
 import numpy as np
 import pandas as pd
 from lxml import html
 
 BASE_PATH = os.path.dirname(__file__)
 
+DATA_SOURCES = {"icubam", "bedcounts", "public", "combined_bedcounts_public"}
+
 DATA_PATHS = {
-  "icu_name_to_department": "data/icu_name_to_department.json",
-  "pre_icubam": "data/pre_icubam_data.csv",
   "department_population": "data/department_population.csv",
   "departments": "data/france_departments.json",
 }
 
 for key, path in DATA_PATHS.items():
   DATA_PATHS[key] = os.path.join(BASE_PATH, path)
 
+
+def load_department_population():
+  return dict(
+    pd.read_csv(DATA_PATHS["department_population"]
+                ).itertuples(name=None, index=False)
+  )
+
+
+def load_france_departments():
+  return pd.read_json(DATA_PATHS["departments"])
+
+
+DEPARTMENTS_GRAND_EST = [
+  "Ardennes", "Aube", "Marne", "Haute-Marne", "Meurthe-et-Moselle", "Meuse",
+  "Moselle", "Bas-Rhin", "Haut-Rhin", "Vosges"
```

Comment:
So in the mid-term all this will be loaded from the DB, right? Maybe let's add an inline comment about it?

Reply:
yes i think we can now use some `icu_region` attribute of `BedCounts` objects no? if so, this is useless and can be removed. good point.